inspector/an-inspector/ForgeModule/src/io/trigger/forge/android/modules/tabs/API.java

```diff
@@ -1,43 +1,43 @@
-package io.trigger.forge.android.modules.tabs;
-
-import com.google.gson.JsonObject;
-
-import java.util.WeakHashMap;
-
-import io.trigger.forge.android.core.ForgeParam;
-import io.trigger.forge.android.core.ForgeTask;
-
-public class API {
-	private static WeakHashMap<String, ModalView> modalViews = new WeakHashMap<String, ModalView>();
-	
-	public static void open(final ForgeTask task) {
-		ModalView modal = new ModalView();
-		modalViews.put(task.callid, modal);
-		modal.openModal(task);
-	}
-	
-	public static void executeJS(final ForgeTask task, @ForgeParam("modal") final String modal, @ForgeParam("script") final String script) {
-		if (modalViews.get(modal) != null) {
-			modalViews.get(modal).webViewProxy.stringByEvaluatingJavaScriptFromString(task, script); // TODO augh
-		}
-	}
-	
-	public static void close(final ForgeTask task, @ForgeParam("modal") final String modal) {
-		if (modalViews.get(modal) != null) {
-			modalViews.get(modal).close();
-			task.success();
-		}
-	}
-
-	public static void addButton(final ForgeTask task, @ForgeParam("modal") final String modal, @ForgeParam("params") final JsonObject params) {
-		if (modalViews.get(modal) != null) {
-			modalViews.get(modal).addButton(task, params);
-		}
-	}
-
-	public static void removeButtons(final ForgeTask task, @ForgeParam("modal") final String modal) {
-		if (modalViews.get(modal) != null) {
-			modalViews.get(modal).removeButtons(task);
-		}
-	}
-}

```

Comment:
I can see no difference here. On creating PRs to public or external repos unnecessary code structure changes should be avoided to keep merge conflicts and review overhead at a minimum.

Reply:
There is something wrong with line endings here not related to auto formatting. I am currently investigating it.

Reply:
I looked into it and the line endings don't affect the "annotate" feature. I would resolve this for now and might change if Trigger Guys complain about it.